### ISSUE TITLE
Fix website generation

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/OSPSuite.GlobalSensitivity#7]
[_pkgdown.yml](https://github.com/Open-Systems-Pharmacology/OSPSuite.GlobalSensitivity/blob/develop/_pkgdown.yml) must be adjusted

**Comment by rengelke** [comment]
@Yuri05 pkgdown workflow is active and `_pkgdown.yml` is in place and correct. 
page build still fails (https://github.com/Open-Systems-Pharmacology/OSPSuite.GlobalSensitivity/actions/runs/28100683502): 
   `"232 | Error: No such file or directory @ dir_chdir0 - /github/workspace/docs"`

I think we have to point to the right branch/directory: Settings -> Pages -> Source: gh-pages, branch / (root).

**Comment by Yuri05** [comment]
@rengelke yep, was in the repo settings. Fixed now (dev site: https://www.open-systems-pharmacology.org/OSPSuite.GlobalSensitivity/dev/ )
